guest/internal/proto/proto.go

```diff
@@ -91,3 +91,41 @@ func (o *Node) Spec() *protoapi.NodeSpec {
 func (o *Node) Status() *protoapi.NodeStatus {
 	return o.Msg.Status
 }
+
```

Comment:
@sanposhiho 
I am not sure this should be defined here again because these are basically same as this.
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/ce05823c9711742f6b02fe79f6d7e3fbeab228d1/guest/internal/prefilter/types.go

Reply:
Rather I think we should remove ↓ then?
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/ce05823c9711742f6b02fe79f6d7e3fbeab228d1/guest/internal/prefilter/types.go

Reply:
Removed Status() in below file:)